Detectors/MUON/MCH/IO/src/tracks-dumper-workflow.cxx

```diff
@@ -9,24 +9,40 @@
 // granted to it by virtue of its status as an Intergovernmental Organization
 // or submit itself to any jurisdiction.
 
-#include <vector>
+#include "DataFormatsMCH/ROFRecord.h"
+#include "DataFormatsMCH/TrackMCH.h"
+#include "DetectorsRaw/HBFUtilsInitializer.h"
 #include "Framework/ConfigParamSpec.h"
-#include "MCHWorkflow/TrackReaderSpec.h"
+#include "SimulationDataFormat/MCCompLabel.h"
+#include "TrackDumperSpec.h"
+#include <fmt/format.h>
+#include <gsl/span>
+#include <iostream>
+#include <vector>
+
+void dump(std::ostream& os, const o2::mch::TrackMCH& t)
+{
+  auto pt = std::sqrt(t.getPx() * t.getPx() + t.getPy() * t.getPy());
+  os << fmt::format("({:s}) p {:7.2f} pt {:7.2f} nclusters: {} \n", t.getSign() == -1 ? "-" : "+", t.getP(), pt, t.getNClusters());
+}
```

Comment:
The dump function and most of the includes are not needed here.
Also I don't think the HBFUtils stuffs are needed here as the dumper is more like a writer than than a reader.
But these are details that can be fixed (or not) later so I approve the PR anyway.

Reply:
thanks.